src/api/app/controllers/webui/monitor_controller.rb

```diff
@@ -126,6 +119,23 @@ def discard_cache?
 
   private
 
+  def get_history(prefix, arch_name, range)
+    gethistory("#{prefix}_#{arch_name}", range, !discard_cache?)
```

Comment:
This is why I miss method overloading... I would get rid of this method and just call this line directly.